### PR TITLE
Update mobile.md

### PR DESCRIPTION
**Pull request by abhi-bhatra**
Mobile: Add mobile interface Infographic in Readme File

Explanation: 
I am using Joplin from a very long time. I visited the Joplin Repo, and I found that Desktop Application Infographic is available which helps user in Navigating the Desktop Application. But, There is no such useful Infographic for Mobile interface.
I created an infographic for Mobile Application, which might help the users a lot.

### CODE REVIEW

**Comment by abhi-bhatra** [comment]
@laurent22 I have added the Mobile UI Infographics to readme/mobile.md which enhances the user experience documentation.
Take a look: 
https://user-images.githubusercontent.com/63901956/127650128-2e00a545-1973-45ed-8442-74f8fc39277e.png

**Comment by tessus** [comment]
afaik graphics should be local to the files and not loaded from external sources.
Let's see what Laurent says...

**Comment by abhi-bhatra** [comment]
Hi
The graphic is created by me itself. It is not loaded from any external resources. I have been using joplin, and its just my own user experience I have shared.

**Comment by tessus** [comment]
I think youu misunderstood. Put the image in the PR. (Check how other images are used in the md files.)

**Comment by laurent22** [comment]
Indeed thank you for the image, I think it's useful, but it needs to be part of the repository. Also please convert it to a JPEG at 90% quality (should result in a much smaller file, but still good quality).

**Comment by abhi-bhatra** [comment]
Thanks @laurent22 and @tessus
I got your point and applied the necessary changes

File: readme/mobile.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Mobile app
+<img src="https://github.com/abhi-bhatra/joplin/blob/patch-3/Assets/JoplinMobileui.jpg" style="max-width=100%"/>
```
Comment (tessus) [inline readme/mobile.md:2]: You are still pointing to your repository.

**Review by tessus — CHANGES_REQUESTED** [review]

File: readme/mobile.md (hunk 1 above)
Comment (tessus) [reply readme/mobile.md:2]: And the file should be renamed to `JoplinMobileUI.jpg` and placed in `Assets/WebsiteAssets/images/`.

**Comment by tessus** [comment]
I mentioned you should check how other images are included. When you check the readme or other md files, including an image usually looks like this:

```
![](https://joplinapp.org/images/imagename.jpg)
```

**Comment by tessus** [comment]
Are vyou sure you saved the image as 90% quality jph. It still seems rather large.

**Comment by abhi-bhatra** [comment]
@tessus Yes I have saved the image with 90% quality. Do, I keep it more small ?

**Comment by tessus** [comment]
> Yes I have saved the image with 90% quality. Do, I keep it more small ?

1.14MB seems a bit large for a few lines and very few colors, but if it's been saved as 90%, it should be ok. 

> I have also changed the image link to laurent22:dev . Kindly review and please guide me on the same!

I'm not sure about this part, since it's a bit inconsistent in the readme and other md files.
@laurent22 shall we keep it as is `` or change it to ``

**Comment by abhi-bhatra** [comment]
@tessus The pull have been disabled to the main `laurent:dev` so I am refering to this `https://github.com/laurent22/joplin/tree/dev/Assets/WebsiteAssets/images/JoplinMobileUI.jpg` and when pull request is merged, it will be address correctly. @laurent22 please guide me on the same!

**Comment by tessus** [comment]
I think you still don't understand what I was saying. If you had looked at the md files, you would have realized that there are 2 options, namely the ones I mentioned above.

Please don't mention us. We get notifications anyway. I only mentioned Laurent to indicate that we can merge as soon as he makes his decision.

**Comment by laurent22** [comment]
For the image, please use the attached one as it's better compressed.

DemoMobile

Link should be like in desktop.md - ``

And the image should go in docs/images

**Comment by tessus** [comment]
> And the image should go in docs/images

Yep, sorry, my bad. I mixed up the dirs.

**Comment by abhi-bhatra** [comment]
> And the image should go in docs/images

Thank you @laurent22 that is why I am confused with how to link the image like this one : `https://joplinapp.org/images/DemoMobile.jpg`
I will do the necessary changes now
Thanks to @tessus too ❤

**Comment by tessus** [comment]
Thanks for the changes.

**Pull request merged**